docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -338,39 +338,95 @@ To set up continuous integration using GitHub Actions, you can use your own Dags
 
----
+After making the above changes and commiting the workflow file, the CI process should be triggered to deploy your GitHub repository to Dagster Cloud. During the deployment, the agent will attempt to load your code and update the metadata in Dagster Cloud. Once finished, you should see the GitHub Action complete successfully and also be able to see the code location under the "Deployment" tag in Dagster Cloud.
 
-The agent will attempt to load your code. After the code is loaded, the new code location will show a **Loaded** status and a sample Dagster job will display in Dagster Cloud.
+---
 
 When finished, [continue to the next step](#step-4-set-up-environment-variables-and-secrets).
 
 </TabItem>
 <TabItem name="Other CI/CD provider">
 
-For continuous integration using a CI/CD provider other than GitHub, your system should:
+For continuous integration using a CI/CD provider other than GitHub, your system should use the `dagster-cloud ci` subcommand to deploy code locations to Dagster Cloud.
+
+Ensure that you have created a `dagster_cloud.yaml` file as described in \[the quickstart]. For detailed documentation about this file see the [dagster_cloud.yaml reference](/dagster-cloud/managing-deployments/dagster-cloud-yaml). The following steps are typically implemented in the CI/CD workflow:
+
+1. Set the build environment variables:
+
+   - `DAGSTER_CLOUD_ORGANIZATION` - \[required] the name of your organization in Dagster Cloud.
+   - `DAGSTER_CLOUD_API_TOKEN` - \[required] a token created in Dagster Cloud. This is a sensitive value and should be stored as a CI/CD secret, if possible.
+   - `DAGSTER_BUILD_STATEDIR` - \[required] path to a blank or non-existent temporary directory on the build machine. This directory is used to store local state during the build.
+
+2. Run the configuration check:
+
+   ```shell
+   dagster-cloud ci check --project-dir=.
+   ```
+
+   This is an optional step but useful to validate the contents of your `dagster_cloud.yaml` and connection to Dagster Cloud.
+
+3. Initialize the build session:
+
+   ```shell
+   dagster-cloud ci init --project-dir=.
+   ```
+
+   This reads the `dagster_cloud.yaml` configuration and initializes the `DAGSTER_BUILD_STATEDIR`.
+
+4. Build and upload Docker images for your code locations.
+
+   The Docker image should contain a Python environment with `dagster`, `dagster-cloud` and your code. For reference, see the [example Dockerfile](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/Dockerfile) in our quickstart repository. The example uses `pip install .` to install the code including the dependencies specified in [setup.py](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/setup.py).
```

Comment:
```suggestion
   The Docker image should contain a Python environment with `dagster`, `dagster-cloud`, and your code. For reference, see the [example Dockerfile](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/Dockerfile) in our quickstart repository. The example uses `pip install .` to install the code including the dependencies specified in [`setup.py`](https://github.com/dagster-io/dagster-cloud-hybrid-quickstart/blob/main/setup.py).
```

- Please use serial/oxford commas
- Adding code formatting to `setup.py`
- Is the period in `pip install .` correct? (I'm not familiar with this, so I'm asking to make sure it's not a typo)

Reply:
Yes the period in `pip install .` is required.